.github/workflows/run-docker-tests-on-garm.yaml

```diff
@@ -0,0 +1,56 @@
+name: CI | Run docker integration tests
+on:
+  workflow_call:
+    inputs:
+      tarball-suffix:
+        required: false
+        type: string
+      commit-hash:
+        required: false
+        type: string
+      target-branch:
+        required: false
+        type: string
+        default: ""
+
+jobs:
+  run-docker-tests:
+    strategy:
+      # We can set this to true whenever we're 100% sure that
+      # the all the tests are not flaky, otherwise we'll fail
```

Comment:
This sentence has a syntax problem "the all the tests". Since this comment appears in other YAML, this can be fixed globally I guess.

Reply:
Fixed now, will fix globally later, thanks!